recipes/fontconfig/all/conanfile.py

```diff
@@ -40,9 +40,12 @@ def requirements(self):
         self.requires("expat/2.2.10")
         if self.settings.os == "Linux":
             self.requires("libuuid/1.0.3")
+        elif self.settings.os == "Macos":
+            self.requires("libgettext/0.20.1")
```

Comment:
I'm not sure, but I suspect that it's required if not Linux.

Reply:
do we have an idea why the test recipe passes on C3I without this ? (the idea would be to improve the test recipe)

Reply:
that might be gettext installed from brew (explicitly or as dependency)